bin/install-php5-xapian.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
```

Comment:
Partly I just wanted to make this as simple as possible and `adding site-specific-install.sh` wasn't helping with that as it only installs under nginx. So yes, it probably could but it was going to be a lot more work than I really wanted to put in to this at the time.